kfs-web/start-tomcat.sh

```diff
@@ -2,4 +2,4 @@
 
 export EXTERNAL_CONFIG="-Dadditional.kfs.config.locations=$UA_KFS_EXTERNAL_CONFIG_DIR/kfs-config.properties,$UA_KFS_EXTERNAL_CONFIG_DIR/kfs-rice-config.properties,$UA_KFS_EXTERNAL_CONFIG_DIR/kfs-security-config.properties,$UA_KFS_INSTITUTIONAL_CONFIG_DIR/institutional-config.properties"
 
-mvn $EXTERNAL_CONFIG -DskipTests=true -P oracle -Dserver.name="http://localhost:8080" tomcat7:run-war
+mvn -DLOG4J_XML_PATH="$HOME/logs" $EXTERNAL_CONFIG -DskipTests=true -P oracle -Dserver.name="http://localhost:8080" tomcat7:run-war
```

Comment:
This log path looks like it would interfere with how other devs have might have their setup. Is this a forgotten change you used for local, but forgot to revert?